contents/swift/welcome-to-swift.md

```diff
@@ -312,7 +350,26 @@ Reference cycles are bad, because they cause memory leaks. Even though A and B a
 
 Here is an [article](https://krakendev.io/blog/weak-and-unowned-references-in-swift) with greater in-depth explanation and examples.
 
-## CocoaPods
+### Benefit: Reuse Existing Code
+
+When you have been working on a Swift project for a while and start to think "Hmm... Someone has probably done something like this before" or "This is a common problem and there should be a library for this", check out [CocoaPods](https://guides.cocoapods.org/using/getting-started.html), the package/library manager for Swift. Apart from using existing library, you may also make your code as a "library" by CocoaPods.
```

Comment:
This part feels odd. You take a conversational tone that is different from how the rest of the article is structured. Why not keep it professional?

CocoaPods is a dependency manager for Swift and Objective-C Cocoa projects that has over 58 thousand libraries and is used in over 3 million apps. Instead of reinventing the wheel, you can check this out to obtain code that helps resolve common issues. If you felt you have done something new with Swift, you can also make your code into a library with CocoaPods.